tests/order.ts

```diff
@@ -1593,121 +1592,6 @@ describe('orders', () => {
 		await clearingHouse.closePosition(marketIndex);
 	});
 
-	it('PlaceAndFill LONG Order multiple fills', async () => {
```

Comment:
order was getting completely filled in one fill, didnt seem worth the effort to fix